README.md

```diff
@@ -74,6 +74,9 @@ curl -L https://install.meilisearch.com | sh
 
 # Launch Meilisearch
 ./meilisearch --master-key=masterKey
+
+# Or with docker (this example saves the data across docker sessions in ../meili_data)
+docker run --rm --name meili -d -p 7700:7700 -v $(pwd)/../meili_data:/meili_data getmeili/meilisearch meilisearch --master-key=secretMasterKey
```

Comment:
Hi @tacman I appreciate your contribution, but all the readmes should have the same structure to improve our ability to automate the changes. Your suggestion will be taken into consideration when we recreate them with the help of the docs team in the future! (the work was made on meilisearch-js already)

Reply:
Understood -- but please include it somewhere in the documentation!   It makes getting started with MeiliSearch so easy.  In fact, I put it there so when I move to a new machine I can remember how to start it.